lib/google/gax/version.rb

```diff
@@ -29,6 +29,6 @@
 
 module Google
   module Gax
-    VERSION = '0.8.0'.freeze
+    VERSION = '0.9.0'.freeze
```

Comment:
I think we might take this opportunity to  force the client library to upgrade to have the http header tracking capability when they upgrade google-gax? for example, all the gemspec should rely on 0.8.1 or 0.9.0?